src/main/java/com/stripe/model/issuing/Settlement.java

```diff
@@ -22,6 +22,10 @@
   @SerializedName("bin")
   String bin;
 
+  /** The date that the transactions are cleared and posted to user's accounts. */
+  @SerializedName("clearing_date")
+  Long clearingDate;
```

Comment:
Would like to flag this, as I didn't see it updated in 7.x

Reply:
yeah they keep changing those and that was not API reviewed and I kind of punted on this one for now since it was coming with autogen